# Zimbabwe Training 1-4 July 2025

Repository to hold notes, files and links for the Zimbabwe training

Home: <https://github.com/seqafrica/zimbabwe-course>


## Programme

 * [Zimbabwe training agenda](Zimbabwe_training_agenda_updated.pdf)

## Day 1

Slides

 * [Day 1 Slides](https://zwets.it/course/slides/SeqAfrica_Bioinformatics_training_2025_DAY1.pdf)
 * [Bioinformatics Basics](https://zwets.it/course/slides/bioinformatics-basics.pdf)


Links

 * [Blog explaining ONT sequencing](https://nanoporetech.com/blog/how-oxford-nanopore-sequencing-works)
 * [Nice animation of ONT sequencing](https://youtu.be/RcP85JHLmnI)
 * [Illumina Sequencing by Synthesis video](https://www.youtube.com/watch?v=fCd6B5HRaZ8)

Exercise links

 * [Mystery species (did we really find Asgard archaea?)](mystery/README.md)
 * [Bacterial Species identification](identify/README.md)

## Day 2

Slides

 * [Day 2 Slides](https://zwets.it/course/slides/SeqAfrica_Bioinformatics_training_2025_DAY2.pdf)
 * [Illumina Reads QC](https://zwets.it/course/slides/illumina-reads-qc.pdf)
 * [FastQC Puzzles](https://zwets.it/course/slides/fastqc-puzzles.pdf)

Links

 * [MinKNOW Report (Land Rover Sequencing)](https://zwets.it/course/landrover/report.pdf)
 * [Nanoplot repository](https://github.com/wdecoster/NanoPlot) and [Nanoplot Online (sequencing summary only)](https://nanoplot.bioinf.be/)
 * [QUAST site](https://quast.sourceforge.net/quast.html) and [WebQuast (currently down)](https://quast.sourceforge.net/webquast.html)

Exercise links

 * (Optional: for command-line experimenters) [Installing WSL (and EPI2ME) on Windows](https://zwets.it/course/epi2me/windows-epi2me-install.pdf)
 * [Perform FastQC on MiSeq reads](fastqc/README.md)
 * [ONT QC - Analysis of NanoPlot reports](nanoplot/README.md)
 * [Reads and assembly QC - MultiQC reports](multiqc/README.md)

## Day 3

Slides

 * [Day 3 Slides](https://zwets.it/course/slides/SeqAfrica_Bioinformatics_training_2025_DAY3.pdf)
 * [BAP Slides](https://zwets.it/course/slides/bap-slides.pdf) and [BAP repository](https://github.com/zwets/cge-bap)
 * [Bioinformatics Recap](https://zwets.it/course/slides/bioinformatics-recap.pdf)

Links

 * [hAMRonization](https://github.com/pha4ge/hAMRonization) and [hAMRonization_workflow](https://github.com/pha4ge/hAMRonization_workflow)

Exercise links

 * [AMR detection and typing with online tools](amr/README.md)
 * [Build your own anything-finder](mydbfinder/README.md)

## Day 4

Slides

 * [Day 4 Slides](https://zwets.it/course/slides/SeqAfrica_Bioinformatics_training_2025_DAY4.pdf)
 * [Phylogeny Slides](https://zwets.it/course/slides/phylogeny-intro.pdf)
 * [Simulated Exercise Instructions](simex/simulated-exercise-instructions.pdf) and [metadata template](simex/Exercise_metadata_not_filled.xlsx)

Links

 * [Enterobase](https://enterobase.warwick.ac.uk/) and [documentation](https://enterobase.readthedocs.io/en/latest) and [the paper](https://www.biorxiv.org/content/10.1101/613554v3)
 * [On-line Grapetree](https://achtman-lab.github.io/GrapeTree/MSTree_holder.html) and [off-line installation instructions](grapetree/README.md), [installation on Windows](grapetree/windows-grapetree-install.pdf)
 * Some demo datasets are linked from [here](grapetree/README.md)
 * Recent paper on cg/wgMLST with ONT: [Accurate and reproducible whole-genome genotyping for bacterial genomic surveillance with Nanopore sequencing data](https://journals.asm.org/doi/10.1128/jcm.00369-25)

Phylogenetics Tutorials

 * [SSI GenEpi practical](https://github.com/ssi-dk/GenEpi-BioTrain_Virtual_Training_7/blob/main/practicals_s2_phylo.md)
 * [BEAST basics](https://beast.community/first_tutorial)
 * [Timetree tutorials](https://treetime.readthedocs.io/en/latest/tutorials/timetree.html)
 * [IQ-TREE](http://www.iqtree.org/doc/Dating)
 
Simulated Exercise

 * Data will be distributed on USB stick; discussion of results is [here](https://zwets.it/course/slides/SeqAfrica_Bioinformatics_training_2025_DAY4_simex_results.pdf)

